### ISSUE TITLE
Components vs Plugins

### ISSUE THREAD

**Issue by gamaanderson** [nguy/artview#182]
Is the separation between components and plugins still relevant? Does it make sense in the way it is implemented today? Should we reorganize it?

**Comment by nguy** [comment]
I agree that it no longer seems to make sense to have the separation. I'm fine with changing this either now or in the future.

**Comment by nguy** [comment]
Come to think of it, I believe we should change this within the next couple of months to have testing time and a clean repo for the August short course.

**Comment by gamaanderson** [comment]
yes, before release 1.3
